apps/ledger-live-desktop/src/newArch/features/MemoTag/components/LearnMoreCta.tsx

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import { Link } from "@ledgerhq/react-ui";
+import { Trans } from "react-i18next";
+import { openURL } from "~/renderer/linking";
+
+type LearnMoreCtaProps = {
+  size?: "small" | "medium" | "large";
+  color?: string;
+  style?: React.CSSProperties;
+  Icon?: React.ComponentType<{ size: number; color?: string }>;
+  url: string;
+};
+
+const LearnMoreCta = ({
+  size = "small",
+  color = "neutral.c80",
+  style,
+  Icon,
+  url,
+}: LearnMoreCtaProps) => {
+  if (!url) return null;
+
```

Comment:
you can use `useLocalizedUrl(url)` to get url for the current locale used

Reply:
done